CODE/Logicytics.py

```
@@ -286,69 +288,93 @@
 
         for thread in threads:
             thread.join()
+    elif ACTION == "performance_check":
+        execution_times = []
+        EXECUTION_LIST = generate_execution_list()
+        for file in range(len(EXECUTION_LIST)):
+            start_time = datetime.now()
```

Ruff (0.8.0)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)